### PR TITLE
It was hard but i did it

### PR DESCRIPTION
**Pull request by VasilPan**
[DEMO LINK]( https://VasilPan.github.io/layout_html-form/)
 [TEST REPORT LINK](https://VasilPan.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,123 @@
...
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action ="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="mainelement">
+        <legend>Personal information</legend>
+        <label  class="one">
+          Surname:
+          <input type="text" name="perinfo"
+          >
```
Comment (SanyaBratashchuk) [inline src/index.html:20]:
```suggestion
          <input type="text" name="perinfo">
```

Comment (SanyaBratashchuk) [reply src/index.html:20]: Fix it everywhere

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,123 @@
...
+        </label>
+        <label name="perinfo" class="one">
+          How old are You?
+          <input type="number"
+          name="perinfo"
+          min="0"
+          max="100"
+          value="12"
+          autocomplete="off"
+          >
```
Comment (SanyaBratashchuk) [inline src/index.html:35]:
```suggestion
          <input 
            type="number"
            name="perinfo"
            min="0"
            max="100"
            value="12"
            autocomplete="off"
          >
```

Comment (SanyaBratashchuk) [reply src/index.html:35]: Fix it everywhere

File: src/style.css [hunk 3]
```diff
@@ -1 +1,9 @@
-/* styles go here */
+.mainelement {
+  margin-bottom: 20px;
+
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:4]:
```suggestion
.mainelement {
  margin-bottom: 20px;
}
```

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,123 @@
...
+        <legend>Additional info</legend>
+        <lable class="one">
+          Comments:
+          <textarea
+          name="add"
+          max="100"
+          main="1"
+        >
+          </textarea>
+        </lable>
```
Comment (SanyaBratashchuk) [inline src/index.html:115]: `lable` ?

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,123 @@
...
+          Surname:
+          <input type="text" name="perinfo"
+          >
+        </label>
+        <label class="one">
+          Name:
+          <input type="text" name="perinfo"
+          >
+        </label>
+        <label name="perinfo" class="one">
```
Comment (SanyaBratashchuk) [inline src/index.html:27]:
```suggestion
        <label name="perinfo" class="label">
```

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,123 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action ="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="mainelement">
```
Comment (SanyaBratashchuk) [inline src/index.html:15]:
```suggestion
      <fieldset class="group">
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
check your code style with [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
add demo and test report link to your pull request

File: src/index.html (hunk 5 above)
Comment (SanyaBratashchuk) [reply src/index.html:27]: fix it

File: src/index.html (hunk 1 above)
Comment (SanyaBratashchuk) [reply src/index.html:20]: Fix it

File: src/index.html (hunk 6 above)
Comment (SanyaBratashchuk) [reply src/index.html:15]: fix it

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Fix previous comments. If you have any questions feel free to ask in fe_chat1 in Slack

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
I don't see any changes since last review

**Pull request closed without merging**